.github/CONTRIBUTING.md

```diff
@@ -83,17 +72,62 @@ yarn link webpack-cli
   yarn build
   ```
 
+
+## Testing
+
+### Using npm
+
 * Run all the tests with:
-  - `yarn test`
+  
+  ```bash
+  npm run test
+  ```
 
-* To test a single CLI (flag) test case:
-    - `BIN_TEST_CASES_GREP=/myCase jest test/BinTestCases.test.js`
+* Test a single CLI test case:
+  
+  ```bash
+  npx jest path/to/my-test.js 
+  ```
 
-* To test a single CLI (other type of) test case:
-    - `yarn jest path/to/my-test.js`
+* You can also install jest globally and run tests without npx:
```

Comment:
I wouldn't suggest to install a package globally. Npx is enough as it comes for free and there's no installation process

Reply:
It has been introduced by #777

Reply:
So why it is marked as an addition in this PR? Did you rebase your branch?

Reply:
It's for people that want to reuse a scaffold @ematipico instead of re-installing every time. I don't think `npx` will completely solve that issue

Reply:
IMO Reusing a scaffold is related to `@webpack-cli/init` being able work with it's global installation @evenstensberg.

I think for this issue we should *just* write if the user has global installation of jest then we should run jest without npm for test.

Reply:
@ematipico I moved this code to separate setup and testing, it is marked as deleted [some lines above](https://github.com/webpack/webpack-cli/pull/768/files#diff-8cc7b2b0d78dd2501610391c086a8516L59)